### PR TITLE
Add quotes to  filename in examples

### PR DESCRIPTION
**Pull request by lparsons**

### CODE REVIEW

File: docs/_writing_parameters.rst [hunk 1]
```diff
@@ -77,7 +77,7 @@ support `Cheetah <http://www.cheetahtemplate.org/docs/users_guide_html/users_gui
 ::
 
     #if $mask_regions
-    -M $mask_regions
+    -M "$mask_regions"
```
Comment (lparsons) [inline docs/_writing_parameters.rst]: Sure. Stylistic preference?

Comment (peterjc) [inline docs/_writing_parameters.rst]: Less risky with regards to special characters - a relatively recent tweak to the best practice guidelines which used to use double quotes: https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#command-formatting https://github.com/galaxy-iuc/standards/commit/25a1a656abea799cd7d3c612201fa53892d97334

Comment (lparsons) [inline docs/_writing_parameters.rst]: BTW, it seems these docs are duplicated... http://planemo.readthedocs.io/en/latest/standards/docs/best_practices/tool_xml.html and https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html Is there a migration taking place?

**Comment by peterjc** [comment]
The TravisCI failure is unrelated,

```
$ flake8 planemo tests
planemo/commands/cmd_travis_init.py:9:1: I100 Import statements are in the wrong order. from planemo should be before from planemo
ERROR: InvocationError: '/home/travis/build/galaxyproject/planemo/.tox/py27-lint/bin/flake8 planemo tests'
```

**Comment by peterjc** [comment]
Thanks Lance!

**Comment by bgruening** [comment]
Thanks!

**Pull request merged**